openshift/log-anomaly-detector.app.yaml

```diff
@@ -129,9 +131,11 @@ objects:
               - mountPath: /opt/app-root/src/${MODEL_DIR}
                 name: model-storage
             resources:
+              request:
```

Comment:
It's `requests` (https://docs.openshift.com/enterprise/3.1/dev_guide/compute_resources.html#dev-cpu-requests)